Makefile

```diff
@@ -10,3 +10,10 @@ generate:
 
 test:
 	go test -count=1 -coverprofile=cover.out ./...
+
+lint:
```

Comment:
I needed a way to run `golangci-lint` in local dev, to reproduce the errors seen in CI.

Reply:
The action is supposed to do things like annotate the code, but I haven’t seen it working so far and haven’t had time to dig in. Something to consider for sure.